### ISSUE TITLE
Process Managment

### ISSUE THREAD

**Issue by lite20** [NodeOS/NodeOS#208]
So we took the first step in developing the to be 'asgard' system in implementing a service starter (PalmTree). One missing ability still remains. On for example Ubuntu we can kill any process with pkill.

Since NodeOS doesn't use any service to persay monitor services, it might be more efficient that we simply implement a process killer like pkill. 

Along with pkill, users may need plist in order to see running processes. With a way to start, check, and stop services and programs, I can't see why NodeOS wouldn't be server capable :)

How would we go about implementing these process managing tools?

**Comment by Coretool** [comment]
Maybe you want to have a look at [ps-node] (https://www.npmjs.com/package/ps-node) to get your process list...

**Comment by Coretool** [comment]
And refering to the bottem of this [so-question] (http://stackoverflow.com/a/1127652) ... To kill a process you can use `process.kill`

**Comment by lite20** [comment]
> @lite20 Maybe you want to have a look at ps-node to get your process list...

Yeah this looks good. As it has no command line we would just have to create one (do you know the status of NodeOS support for blessed?)

> And refering to the bottem of this stack overflow answer ... To kill a process you can use process.

With these I guess I'll get started on a command-line tool for NodeOS to handle these. I'll name it asgard in honor of asgard.

**Comment by piranna** [comment]
We can use [ndm](https://github.com/npm/ndm), it's just a matter of add support for our JSON config file. They are currently using templates, so first we should allow to use render functions to make it cleaner and easier... seems there has been some progress towards it.

**Comment by lite20** [comment]
> We can use ndm, it's just a matter of add support for our JSON config file. They are currently using templates, so first we should allow to use render functions to make it cleaner and easier... seems there has been some progress towards it.

Okay! I'll go take a peep

**Comment by piranna** [comment]
:+1:
